SentinelOne/sentinelone/ingest/parser.yml

```
@@ -541,7 +541,14 @@ stages:
           sentinelone.data.uploadedFilename: "{{json_event.message.data.uploadedFilename}}"
           sentinelone.data.uuid: "{{json_event.message.data.uuid}}"
 
-          user.name: "{{json_event.message.data.userName or json_event.message.data.username or json_event.message.data.lastLoggedInUserName}}"
+          user.name: "{{json_event.message.data.lastLoggedInUserName}}"
+
+      - set:
+          user.domain: "{{json_event.message.data.sourceprocessusername.split('\\\\')[0]}}"
+          user.name: "{{json_event.message.data.sourceprocessusername.split('\\\\')[1]}}"
```

IMHO, `process.user.*` are more suitables:
```suggestion
          process.user.domain: "{{json_event.message.data.sourceprocessusername.split('\\\\')[0]}}"
          process.user.name: "{{json_event.message.data.sourceprocessusername.split('\\\\')[1]}}"
```